bibtex-actions-file.el

```diff
@@ -165,9 +167,13 @@ use 'orb-edit-note' for this value."
             (file-exists (file-exists-p file)))
       (funcall bibtex-actions-file-open-function file)
     (let* ((uuid (org-id-new))
-           (title (bibtex-actions-get-value "title" (bibtex-actions-get-entry key)))
+           (entry (bibtex-actions-get-entry key))
+           (note-title
+            (bibtex-actions--format-entry-no-widths
+             entry
+             (bibtex-actions-get-template 'note)))
            (content
-            (concat ":PROPERTIES:\n:ID:  " uuid "\n:END:\n#+title: Notes on " title "\n")))
+            (concat ":PROPERTIES:\n:ID:  " uuid "\n:END:\n" note-title "\n")))
```

Comment:
There's an issue here:

Right now, I have half of this which is handled via template, and the other via hard-coded function logic.

It might be better to allow the template a `uuid-new` or some such field, so the template could include of it:

```elisp
 ":PROPERTIES:\n:ID: ${uuid-new}\n:END:\n#+title: Notes on ${author editor}, ${title}"
```

Reply:
Maybe better to use something like this, to remove the `org-id` dependency?

http://ergoemacs.org/emacs/elisp_generate_uuid.html